resources/web/docs_js/components/console_widget.jsx

```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
   }
 }
 
+export class _TryConsoleSelector extends Component {
+  componentWillMount() {
+    const defaultVals = omit(['langStrings', 'saveSettings', 'url_label', 'setting'], this.props);
+    this.setState(defaultVals);
+  }
+
+
+  render(props) {
+    const handleConfigureClick = (e) => {
+      e.preventDefault();
+      props.settingsModalAction();
+    }
+
+    return (
+      <div className="try_console_selector">
+        <h4>Try in Console</h4>
+        <p>We were unable to detect a running console server.</p>
+        <p>
+          <a
+            id="try_console_selector_try_cloud_button"
+            className="button btn-primary btn-small"
+            href="https://cloud.elastic.co/registration"
+            target="_blank"
+          >
+            Sign up for Elastic Cloud trial
+          </a>
+        </p>
+        <p>
+          <a
+            id="try_console_selector_install_elasticsearch_button"
+            href="https://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html"
```

Comment:
```suggestion
            href="https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html"
```

I think this is a better link